src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/service/NetworkWatcherService.java

```diff
@@ -206,14 +213,24 @@ private void timeoutAction(Packet packet) {
         if (confirmedPackets.remove(packet)) {
             log.debug("Detect discovery packet lost sent via {} id:{} task:{}",
                       packet.endpoint, packet.packetNo, taskId);
-            carrier.discoveryFailed(packet.getEndpoint(), packet.packetNo, now());
+            if (active) {
```

Comment:
`timeoutAction` called only from `tick(...)`.

Reply:
This condition has been removed.